### ISSUE TITLE
Document information about COM Registration Fixing Tool.

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#8962]
In NVDA's tools menu, one of the options is "Run COM Registration Fixing Tool".

This tool is not documented in the User Guide (less importantly, I can't see it listed anywhere in the what's new file or issues either, although running the tool is suggested in several issues and I advise people to use it in attempting to fix several issues).

**Comment by Adriani90** [comment]
@Qchristensen could you please suggest a description for it including the details relevant for users? Maybe Mick or Reef could give some background on it as well. I could add it to the user guide but I want to check with you first before adding wrong details.

**Comment by Qchristensen** [comment]
I wrote several versions, each one getting too complicated.  So I went back to basics and didn't try to explain the technicalities, just the main causes and symptoms:

"When installing or uninstalling software, sometimes files (particularly DLL files) used by multiple programs can be inadvertently de-registered.  One of the most common symptoms of this is NVDA reporting "unknown" instead of window information.  The COM registration fixing tool ensures that DLLs which are present on the system are correctly registered."
